docs/DeveloperGuide/Tutorials/AddV2I/AssignLaneletIDs/index.md

```diff
@@ -0,0 +1,23 @@
+# Assign Lanelet2 WayID and RelationID to TrafficLight object
```

Comment:
In general I think the whole chapter should be called `Add V2I` and it shouldn't be divided into 2 pages but everything should be here in 3 sections.
ps. it should be clear to the reader what is needed to work and why (I encourage you to describe it in the `UserGuide`)